tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -340,6 +340,17 @@ TangoRosNode::TangoRosNode() : run_threads_(false) {
   } catch (const image_transport::Exception& e) {
     LOG(ERROR) << "Error while creating image transport publishers" << e.what();
   }
+  // Initialize start_of_service_T_area_description as identity.
+  start_of_service_T_area_description_.transform.translation.x = 0.;
+  start_of_service_T_area_description_.transform.translation.y = 0.;
+  start_of_service_T_area_description_.transform.translation.z = 0.;
+  start_of_service_T_area_description_.transform.rotation.x = 0.;
+  start_of_service_T_area_description_.transform.rotation.y = 0.;
+  start_of_service_T_area_description_.transform.rotation.z = 0.;
+  start_of_service_T_area_description_.transform.rotation.w = 1.;
```

Comment:
(minor comment) At least in rosjava, there are some static functions that return a initialized vector / quaternion for some particular cases such as the unit quaternion or the null vector. Perhaps they exist in the Cpp libraries too; it just makes the code a bit shorter.